content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
    Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
    that no driver is selected for those printers.
 
+Epson special case
+~~~~~~~~~~~~~~~~~~
+
+When printing a receipt with an Epson printer, the command that is used is called `GS v 0`. However,
+there are some Epson printers models that do not support this command, they include the following
+models:
+
+- TM-U220
+- TM-U230
+- TM-P60
+- TMP-P60II
+
+If the :abbr:`POS (Point of Sale)` system is using one of these printer models, a particular
+process must be undertaken in order to use another command; `ESC *`.
+
+Process to force ESC * command
+******************************
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `documentation on Epison's site
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for compatible
```

Comment:
```suggestion
   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for `GS v 0` compatible
```

Reply:
I'm afraid that if client see that it looks the same, they will assume it is the same link

Reply:
Noted,  I've made the change.